tests/neb/ui/test_get.py

```diff
@@ -126,3 +126,20 @@ def test_get_col_minor_version(neb_env, col_id, col_version, snapshot):
     ) as zip_dir:
         # THEN the complete zip is downloaded and matches the snapshot
         snapshot.assert_file_or_dir_match(zip_dir, snapshot_name)
+
+@markers.neb
+@markers.nondestructive
+@markers.parametrize("col_id,col_version", [("col11562", "1.19.6")])
+def test_get_col_with_resources(neb_env, col_id, col_version, snapshot):
```

Comment:
I'm curious about this test case. I understand that it differs from the preceding test cases in that it also downloads the resource files. However, does this exercise a new code path on archive (I'm assuming that's the intent -- if the goal is to exercise a new code path on `neb`, then ignore this comment)? 

Specifically, I believe `neb` will use [`/resources/{hash}`](https://github.com/openstax/cnx-archive/blob/027dd59d5ae0dc4045945ea416269487929d550a/cnxarchive/__init__.py#L66) for both [content files](https://github.com/openstax/nebuchadnezzar/blob/6aa1d2d3dbc260b82c38f402623c784506a9f85b/nebu/cli/get.py#L306) and  [resource files](https://github.com/openstax/nebuchadnezzar/blob/6aa1d2d3dbc260b82c38f402623c784506a9f85b/nebu/cli/get.py#L326). In archive, I believe both types of invocations lead to the [same code being exercised](https://github.com/openstax/cnx-archive/blob/027dd59d5ae0dc4045945ea416269487929d550a/cnxarchive/views/resource.py#L32). Am I missing something?